packages/types/src/profile.ts

```diff
@@ -1,9 +1,14 @@
-export interface Profile {
+export interface ParsedIniSection {
```

Comment:
How about 
```suggestion
export interface IniSection {
```

The JavaScript object and interface already indicates the INI file is parsed.